lib/language_pack/ruby.rb

```diff
@@ -14,7 +14,7 @@ class LanguagePack::Ruby < LanguagePack::Base
   NAME                 = "ruby"
   LIBYAML_VERSION      = "0.1.6"
   LIBYAML_PATH         = "libyaml-#{LIBYAML_VERSION}"
-  BUNDLER_VERSION      = "1.17.3"
+  BUNDLER_VERSION      = env("CUSTOM_BUNDLER_VERSION") || "1.17.3"
```

Comment:
Why not use `BUNDLER_VERSION` as the environment variable name? Does that have special meaning?

Reply:
We can. I just didn't want to give the impression that this is a special environment variable that bundler itself cares about.

Reply:
I think that makes sense. bundler does prefix all of it's blessed environment variable names with `BUNDLER_`

Reply:
I'm not sure I see the problem. We're specifying the version of Bundler we want to use. It's not clear what's "custom" about it. Relatedly, I wouldn't expect Bundler to care about a `BUNDLER_VERSION` environment variable in the same way I wouldn't expect Ruby to care about a `RUBY_VERSION` environment variable. By the time you're running either of them, it's too late to be specifying versions.